scripts/mongo-index.js

```diff
@@ -0,0 +1,11 @@
+db.getCollection('actions').createIndex(
+    {
+      'organizer.name.en': "text",
+      'organizer.lastName.en': "text",
+      'organizer.email': "text",
+      'name.en': "text",
+      'description.en': "text",
+      'linkagesDescription.en': "text",
+      'progressMeasured.en':"text"
+    }
+  )
```

Comment:
thi shld be in gaia initialize

Reply:
yes, I could not think of a spot for it at the time so I just put it there to saving.